buildtest/cli/__init__.py

```diff
@@ -430,9 +453,17 @@ def unittest_menu(subparsers):
         subparsers (argparse._SubParsersAction): Subparser object to add subparser
     """
 
-    unittests_parser = subparsers.add_parser(
-        "unittests", help="Run buildtest unit tests", aliases=["test"]
-    )
+    # Subcommands that do not need to be shown in --help
+    h_unittests = {
```

Comment:
I understand what you are doing and this can be improved if we just have one dictionary where keys are the actual commands we can declare everything once and just call the method once as pose to calling it multiple times.

This will require a refactoring for method `add_hidden_parser` since dictionary lookup will be changed.

Reply:
You're right, that would be more efficient.